flow/Makefile

```diff
@@ -636,7 +636,7 @@ finish: $(LOG_DIR)/6_report.log \
         $(RESULTS_DIR)/6_final.sdc \
         $(GDS_FINAL_FILE)
 	@printf "%-25s %10s\n" Log "Elapsed seconds"
-	@grep Elapsed $(abspath $(LOG_DIR))/*.log | awk -F'[:. ]' '{sub(".*+/","",$$1); sub(".log:","",$$1); sub("\\:","",$$1); printf "%-25s %10s\n", $$1, $$6 * 60 +$$7}'
+	@grep Elapsed $(abspath $(LOG_DIR))/*.log | awk 'match($$0, /[a-z_0-9/]+\/([a-z_0-9]+)\.log:Elapsed time: ([0-7]+:)?([0-7]+):([0-7]+)/ , arr) { printf "%-25s %10d\n", arr[1], arr[2]*3600 + arr[3] * 60 + arr[4] }'
```

Comment:
Why is it [0-7]+ ?

Reply:
I'm a bit lazy, I could have written ```[1-7]?[0-7]*``` maybe?

Reply:
1:09 wouldn't match and I'm wondering why not?  It seems this would be simpler if you just added %e to the message and didn't do any math.

Reply:
Ugh.... I was cognitively impared yesterday, it would seem. I have no idea why I used 0-7, mentally I was thinking 0-9...

Reply:
Two test-cases locally:

```
$ echo "foo/bar/test.log:Elapsed time: 1:09:05.39[h:]min:sec."  | awk 'match($0, /[a-z_0-9/]+\/([a-z_0-9]+)\.log:Elapsed time: ([0-9]+:)?([0-9]+):([0-9]+)/ , arr) { printf "%-25s %10d\n", arr[1], arr[2]*3600 + arr[3] * 60 + arr[4] }'
test                            4145
oyvind@corona:~/ascenium/OpenROAD-flow-scripts$ echo "foo/bar/test.log:Elapsed time: 1:09.39[h:]min:sec."  | awk 'match($0, /[a-z_0-9/]+\/([a-z_0-9]+)\.log:Elapsed time: ([0-9]+:)?([0-9]+):([0-9]+)/ , arr) { printf "%-25s %10d\n", arr[1], arr[2]*3600 + arr[3] * 60 + arr[4] }'
test                              69
```

Reply:
[a-z_0-9/]+\/([a-z_0-9]+)  doesn't match 5_2_TritonRoute.log due to the upper case letters.

Reply:
Fixed.

Good catch.

```
echo "lkajsdf/TritonRoute.log:Elapsed time: 1:2:3 " | awk 'match($0, /[a-zA-Z_0-9/]+\/([a-zA-Z_0-9]+)\.log:Elapsed time: ([0-9]+:)?([0-9]+):([0-9]+)/ , arr) { printf "%-25s %10d\n", arr[1], arr[2]*3600 + arr[3] * 60 + arr[4] }'
TritonRoute                     3723
```